packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -41,56 +52,39 @@ export class ProfilingPlugin {
       if (!span) {
         return
       }
-      stackPop(this.compiler, span)
-    })
-  }
-
-  traceLoopedHook(spanName: string, startHook: any, stopHook: any) {
-    let span: Span | undefined
-    startHook.tap(pluginName, () => {
-      if (!span) {
-        span = stackPush(this.compiler, spanName)
-      }
-    })
-    stopHook.tap(pluginName, () => {
-      stackPop(this.compiler, span)
+      span.stop()
     })
   }
 
   traceTopLevelHooks(compiler: any) {
     this.traceHookPair(
-      'webpack-compile',
-      compiler.hooks.compile,
-      compiler.hooks.done,
-      () => {
-        return { name: compiler.name }
-      },
-      (span) => spans.set(compiler, span)
-    )
-    this.traceHookPair(
-      'webpack-prepare-env',
-      compiler.hooks.environment,
-      compiler.hooks.afterEnvironment
```

Comment:
We don't want environment and afterEnvironment anymore?

Reply:
Correct 👍 it's only a few microseconds